mail_tracking/views/res_partner_view.xml

```diff
@@ -6,7 +6,6 @@
 <record model="ir.ui.view" id="view_partner_form">
     <field name="name">Partner Form with tracking emails</field>
     <field name="model">res.partner</field>
-    <field name="sequence">46</field>
```

Comment:
Why is required this change?

Reply:
because of the warning at runbot `includes unknown fields: sequence`
https://runbot3.odoo-community.org/runbot/static/build/3342080-11-0-5f58ef/logs/job_20_test_all.txt

Reply:
I just saw ir.ui.view model and it has defined a field called priority
That is the new sequence
I think so